### ISSUE TITLE
Sample config should provide calendar facilitator

### ISSUE THREAD

**Issue by dveeden** [etsy/morgue#92]
With a copy of the sample config I get this:

```
exception 'ErrorException' with message 'Undefined index: facilitator' in /home/dvaneeden/morgue/features/calendar/lib.php:12
Stack trace:
```

Adding a facilitator entry to the calendar in the config fixed this.

**Comment by dveeden** [comment]
This also fixed another issue I had: the "Click here to make changes" button to edit an pm didn't work

**Comment by mmercedes** [comment]
Good catch!
Fixed bd8903f6dc2c9b1906be7b820dcbf4bc4882e020
